Automattic/jetpack
# Payments blocks: Add more robust set of keywords

Fixes #22898

#### Changes proposed in this Pull Request:

Add a more extensive set of keywords to some of the payments blocks, to
make it easier for users to find what they are looking for.

#### Jetpack product discussion



See paYJgx-1Hh-p2#comment-2156

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






##### Setup
**jetpack**: Build 
**wpcom**: Patch sandbox as usual
**Atomic**: Use jetpack beta plugin to select this branch

##### Test 
 1. Open a new post
 2. Type the keywords
 3. See blocks appear
